apps/docs/content/guides/ai/hybrid-search.mdx

```diff
@@ -0,0 +1,228 @@
+
+## How to combine search methods
+
+Hybrid search merges keyword search and semantic search, but how does this process work?
+
+First, each search method is executed separately. Keyword search, which involves searching by specific words or phrases present in the content, will yield its own set of results. Similarly, semantic search, which involves understanding the context or meaning behind the search query rather than the specific words used, will generate its own unique results.
+
+Now with these separate result lists available, the next step is to combine them into a single, unified list. This is achieved through a process known as “fusion”. Fusion takes the results from both search methods and merges them together based on a certain ranking or scoring system. This system may prioritize certain results based on factors like their relevance to the search query, their ranking in the individual lists, or other criteria. The result is a final list that integrates the strengths of both keyword and semantic search methods.
+
+## Reciprocal Ranked Fusion (RRF)
+
+One of the most common fusion methods is Reciprocal Ranked Fusion (RRF). The key idea behind RRF is to give more weight to the top-ranked items in each individual result list when building the final combined list.
+
+In RRF, we iterate over each record and assign a score (noting that each record could exist in one or both lists). The score is calculated as 1 divided by that record's rank in each list, summed together between both lists. For example, if a record with an ID of `123` was ranked third in the keyword search and ninth in semantic search, it would receive a score of $\dfrac{1}{3} + \dfrac{1}{9} = 0.444$. If the record was found in only one list and not the other, it would receive a score of 0 for the other list. The records are then sorted by this score to create the final list. The items with the highest scores are ranked first, and lowest scores ranked last.
+
+This method ensures that items that are ranked high in multiple lists are given a high rank in the final list. It also ensures that items that are ranked high in only a few lists but low in others are not given a high rank in the final list. Placing the rank in the denominator when calculating score (ie. reciprocal rank) helps penalize the low ranking records.
+
+### Smoothing constant `k`
+
+To prevent extremely high scores for items that are ranked first (since we're dividing by the rank), a `k` constant is often added to the denominator to smooth the score:
+
+$\dfrac{1}{k+rank}$
```

Comment:
Here is a [KaTeX](https://katex.org/) example (supported via the new markdown plugins). Fyi, Notion uses KaTeX as well for their math equations, so these can be copy-pasted into guides now.